torch/_inductor/test_operators.py

```diff
@@ -0,0 +1,24 @@
+import torch.library
+from torch import Tensor
+from torch.autograd import Function
+
+_test_lib_def = torch.library.Library("_inductor_test", "DEF")
+_test_lib_def.define("realize(Tensor self) -> Tensor")
+
+_test_lib_impl = torch.library.Library("_inductor_test", "IMPL")
+for dispatch_key in ("CPU", "CUDA", "Meta"):
```

Comment:
Given that you are doing an autograd-aware implementation, you should register this to `AutogradCPU`, `AutogradCUDA` and `AutogradMeta`.

Reply:
My bad I misread this. I though the custom Function was registered on these keys.